cmd/rook/ceph/admission.go

```diff
@@ -36,11 +45,15 @@ func init() {
 
 func startAdmissionController(cmd *cobra.Command, args []string) {
 	rook.SetLogLevel()
-
 	rook.LogStartupInfo(admissionCmd.Flags())
 
-	context := rook.NewContext()
-
-	a := admission.New(context, "ceph", operator.ValidateCephResource)
-	a.StartServer()
+	err := cephv1.AddToScheme(scheme)
+	if err != nil {
+		rook.TerminateFatal(err)
+	}
+	a := operator.NewAdmissionController("ceph", port, certDir, scheme)
```

Comment:
Do we even need to pass any of these parameters? What if we just define them directly inside StartServer()? It looks like they aren't configurable, so we don't need to expose them here.

Reply:
Moved to server.go